### ISSUE TITLE
[Samples] [Azure.AI.FormRecognizer] Suggest change sample code method to async.

### ISSUE THREAD

**Issue by wantedfast** [Azure/azure-sdk-for-net#11668]
Section [Link](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer#recognize-receipts)

Details:
![image](https://user-images.githubusercontent.com/25993729/80580808-bb1bfb00-8a3e-11ea-9879-57e22cefc65a.png)

Comments: Use async method might be good there.
```
Response<IReadOnlyList<RecognizedReceipt>> receipts = await client.StartRecognizeReceiptsAsync(stream).WaitForCompletionAsync();
```

**Comment by tzhanl** [comment]
@jongio Please review.

**Comment by jongio** [comment]
@annelo-msft

**Comment by annelo-msft** [comment]
We can add async samples where appropriate.  @kinelski, @maririos, @mayurid

**Comment by maririos** [comment]
Anne, should we change it for this specific case? We are doing a sync LRO call (sounds weird) but following the recommendations of showcasing our sync APIs.

**Comment by annelo-msft** [comment]
Related to: https://github.com/azure/azure-sdk-for-net/issues/11475

**Comment by kinelski** [comment]
Fixed by #13224.
